cli/src/commands/squash.rs

```diff
@@ -176,7 +177,10 @@ pub(crate) fn cmd_squash(
     )? {
         let mut commit_builder = squashed.commit_builder.detach();
         let new_description = match description {
-            SquashedDescription::Exact(description) => description,
+            SquashedDescription::Exact(description) => {
+                commit_builder.set_description(description);
+                add_trailers(ui, &tx, &commit_builder)?
+            }
             SquashedDescription::UseDestination => destination.description().to_owned(),
             SquashedDescription::Combine => {
```

Comment:
Did you intentionally make `::Combine` not generate trailers? Can you add comment why?

I can see it's unclear what we should do here, but the behavior seems inconsistent. Depending on the rationale, it might not make sense that `jj split`/`squash` generate new trailers.

Reply:
That was kind of intentional: I didn't see how to integrate the trailers nicely into the combined message.

Now that I rethink about it, we could add a last section in the combined messages with the trailers generated with the `commit_trailers` template that were not found in any other message. That way, the user could decide what to do with them.

Something like

~~~
JJ: Enter a description for the combined commit.
JJ: Description from the destination commit:
message  1

JJ: Description from source commit:
message 2

JJ: Description from source commit:
message 3

JJ: Trailers not found in the squashed commit descriptions:

Signed-off-by: Alice <alice@example.com>

JJ: Lines starting with "JJ:" (like this one) will be removed.
~~~

Reply:
> Now that I rethink about it, we could add a last section in the combined messages with the trailers generated with the commit_trailers template that were not found in any other message.

Sounds reasonable. Can you leave a TODO comment (or just implement it if that's easy)?